apps/api/src/schema/programs.ts

```diff
@@ -1,6 +1,28 @@
 import { z } from "@hono/zod-openapi";
 
 const programIdBase = z.string({ required_error: "programId is required" });
+const majorIdBase = z.string();
+
+export const majorQuerySchema = z.object({
+  majorId: majorIdBase.optional().openapi({
+    description: "The ID of the specific major being requested for",
```

Comment:
"being requested for" is awkward, could be more concise: "The ID of a single major to request, if provided" or something along those lines is most concise.